frontend/src/metabase/visualizations/visualizations/PieChart/PieChart.jsx

```diff
@@ -371,7 +371,8 @@ export default class PieChart extends Component {
         jsx: true,
         majorWidth: 0,
         number_style: "percent",
-        decimals,
+        minimum_fraction_digits: decimals,
+        maximum_fraction_digits: decimals,
```

Comment:
Are we still expecting snake cased named options here?

Reply:
We can use either, since they get normalized, so we may as well use camelCase.